data/huion-gc610-710.tablet

```diff
@@ -0,0 +1,26 @@
+# This tablet description file has been generated using an automated tool:
+# https://github.com/JoseExposito/uclogic-libwacom-tablet-descriptions
+# If you own this tablet, please improve it.
+
+[Device]
+Name=Huion GC610/710
+ModelName=
+Class=Bamboo
+DeviceMatch=usb|256c|006e||HUION_T166;usb|256c|006d||HUION_T166;
+Width=
+Height=
+Layout=
```

Comment:
See #661 and #660, these should get over this test case failure. Not super-happy about #661 (removing the need for a layout file) but if it gets us more tablet support it's still a net win.

Reply:
If an empty `Layout=` entry causes issues, I can delete it. The files are generated automatically, so it'd be a matter of dropping a line of code.

About #661, I added [a comment](https://github.com/linuxwacom/libwacom/pull/661#pullrequestreview-2014070578) suggesting a possible ~solution~ enhancement.

Reply:
Converted to draft as there might be a way to auto generate SVGs. I'm not sure about `Width`/`Height` yet... But I can always google 60 tablet sizes :sleepy:

Reply:
I'd say just add 9x6 and a comment and we're good. *Nothing* reads those values but for backwards compat (and in case someone will want them in the future) we still need to set them. If they're wrong, no-one's going to notice for the foreseeable future.

Reply:
I (finally) added your files for all the description files. Resolving this conversation ;)